distribution/src/main/resources/docker-compose/wiremock/hxinsight/__files/agents.json

```diff
@@ -1,23 +1,45 @@
 [
   {
-    "agentId": "5fca2c77-cdc0-4118-9373-e75f53177ff8",
+    "id": "5fca2c77-cdc0-4118-9373-e75f53177ff8",
+    "hxEnvironmentId": "ecf13dd4-c061-462b-8122-e3203cc40a0d",
     "name": "HR Policy Agent",
+    "version": "1",
     "description": "This agent is responsible for HR policy predictions",
-    "personality": "Keep responses professional and concise",
-    "model": "meta-llama3-8b",
-    "allowedSources": [
+    "instructions": "Keep responses professional and concise",
+    "modelName": "meta-llama3-8b",
+    "sourceIds": [
       "HR Policy Repository"
-    ]
+    ],
+    "permissions": [
+      {
+        "principalType": "user",
+        "id": "e2b9471c-d869-48b4-94ce-5112dd71001c"
+      }
+    ],
+    "avatarUrl": "https://s3.amazonaws.com/avatars/ecf13dd4-c061-462b-8122-e3203cc40a0d.jpg",
+    "staticFilter": {},
+    "dynamicFilter": {}
```

Comment:
where do these come from? I didn't see them on the HX Swagger UI

Reply:
Please see in the `agentResponse` schema. 
See also: https://github.com/HylandSoftware/hxai-data-knowledge-retrieval/blob/539fc9efa9b66cd4fe2340745b5d8bfbdc05a64e/openapi/hxai-discovery-api.yaml

Reply:
Can't access this link but I'll trust you on that.

@mpichura do you have access to the HylandSoftware repositories? In that case, what's the procedure we have to go through to also request access?

Reply:
I think you need to be a part of https://github.com/orgs/HylandSoftware/teams/all-employees team. You can ask [Jakub Jeszke] (https://hyland.enterprise.slack.com/team/U039QEUJ6CU) about that

Reply:
@dsibilio - sorry for late reply - I somehow missed that message. As Krystian mentioned - you can ask Jakub for access. There's also SwaggerUI endpoint available http://hxai-data-platform-dev-swagger-ui.s3-website-us-east-1.amazonaws.com/?urls.primaryName=Insight%20Discovery%20API%20Draft#/Discovery-Agent-API/list_agents, however, I am not sure how up to date that one is (those 2 fields are included there though).